includes/Core/Storage/Data_Encryption.php

```diff
@@ -81,21 +83,36 @@ public function encrypt( $value ) {
 	 *
 	 * @param string $raw_value Value to decrypt.
 	 * @return string|bool Decrypted value, or false on failure.
+	 * @throws \Exception If the value fails to decode.
 	 */
 	public function decrypt( $raw_value ) {
-		if ( ! extension_loaded( 'openssl' ) ) {
+		if ( ! extension_loaded( 'openssl' ) || ! is_string( $raw_value ) ) {
 			return $raw_value;
 		}
 
-		$raw_value = base64_decode( $raw_value, true );
+		$decoded_value = base64_decode( $raw_value, true );
+
+		if ( false === $decoded_value ) {
+			if ( 'development' === Build_Mode::get_mode() ) {
+				throw new \Exception(
+					sprintf(
+						/* translators: %s: raw value */
+						__( 'Failed to decode base64 value. Raw value: %s', 'google-site-kit' ),
+						$raw_value
+					)
```

Comment:
We don't need to translate code errors like this, it's fine to just output them as strings 🙂

Reply:
@tofumatt Due to the phpcs error preventing the commit to pass, I rather included it then adding ignore comment.